wolvic/wolvic_content_browser_client.h

```diff
@@ -36,6 +30,8 @@ class WolvicContentBrowserClient : public ContentBrowserClient {
   content::BrowserContext* GetBrowserContext();
 
   // ContentBrowserClient overrides.
+  std::string GetUserAgent() override;
+  blink::UserAgentMetadata GetUserAgentMetadata() override;
```

Comment:
Both of these methods could be declared as `const`.

Reply:
No, since the parent class declaration doesn't have a const qualifier. (C++ disallows changing constness of methods when overriding)